lang/strings_english.txt

```diff
@@ -1314,6 +1314,7 @@ $s_advanced_filters = 'Advanced Filters';
 $s_simple_filters = 'Simple Filters';
 $s_monitored_by = 'Monitored By';
 $s_attachments = 'attachment(s)';
+$s_files = 'File(s)';
```

Comment:
Why a new string, and not reuse the existing $s_attachments ?

Reply:
We can't use $s_attachments due to casing. It is used in cases that expect lower case. We will need a new string which can be `$s_files = 'File(s)'` or `$s_attachments_capitalized = `Attachment(s)` depends on the string we want.